modules/quotas-requiring-explicit-quota.adoc

```diff
@@ -8,44 +8,100 @@
 +
 [source,yaml]
 ----
-admissionConfig:
-  pluginConfig:
-    ResourceQuota:
-      configuration:
-        apiVersion: resourcequota.admission.k8s.io/v1alpha1
-        kind: Configuration
-        limitedResources:
-        - resource: persistentvolumeclaims <1>
-        matchContains:
-        - gold.storageclass.storage.k8s.io/requests.storage <2>
-----
-<1> The group/resource to whose consumption is limited by default.
-<2> The name of the resource tracked by quota associated with the group/resource to
-limit by default.
-+
-In the above example, the quota system intercepts every operation that
-creates or updates a `PersistentVolumeClaim`. It checks what resources understood
-by quota would be consumed, and if there is no covering quota for those resources
-in the project, the request is denied.
-+
-In this example, if a user creates a `PersistentVolumeClaim` that uses storage
-associated with the gold storage class, and there is no matching quota in the
-project, the request is denied.
+- apiVersion: v1
+  kind: ResourceQuota
+  metadata:
+    name: storage-consumption
+  spec:
+    hard:
+      persistentvolumeclaims: "10" <1>
+      requests.storage: "50Gi" <2>
+      gold.storageclass.storage.k8s.io/requests.storage: "10Gi" <3>
+      silver.storageclass.storage.k8s.io/requests.storage: "20Gi" <4>
+      silver.storageclass.storage.k8s.io/persistentvolumeclaims: "5" <5>
+      bronze.storageclass.storage.k8s.io/requests.storage: "0" <6>
+      bronze.storageclass.storage.k8s.io/persistentvolumeclaims: "0" <7>
+----
+<1> The total number of persistent volume claims in a project.
+<2> Across all persistent volume claims in a project, the sum of storage requested cannot exceed this value.
+<3> Across all persistent volume claims in a project, the sum of storage requested in the gold storage class cannot exceed this value.
+<4> Across all persistent volume claims in a project, the sum of storage requested in the silver storage class cannot exceed this value.
+<5> Across all persistent volume claims in a project, the total number of claims in the silver storage class cannot exceed this value.
+<6> Across all persistent volume claims in a project, the sum of storage requested in the bronze storage class cannot exceed this value. When this is set to 0, it means bronze storage class cannot request storage.
+<7> Across all persistent volume claims in a project, the sum of storage requested in the bronze storage class cannot exceed this value. When this is set to 0, it means bronze storage class cannot create claims.
+
+. Create a new project request template from the modified `template.yaml` file, in the `openshift-config` namespace. By default, the template will be called `project-request`:
++
+----
+$ oc create -f template.yaml -n openshift-config
+----
+
+. Reference the new project request template in the cluster's project configuration resource:
+.. Access the project configuration resource for editing:
++
+** Using the web console:
```

Comment:
> ! using n
To avoid ambiguity, replace this gerund with either “by using” or “that uses.”

Reply:
Resolved.